src/workers/nlmExtractTables.ts

```diff
@@ -38,18 +38,22 @@ const extractTextViaVisionAPI = async (
       },
       {
         role: 'user',
-        content: `I have a PDF with couple of tables related to a company's CO2 emissions. Can you extract the text from screenshot. I will send you the screenshot extract the header and table contents and ignore the surrounding text if they are not related to the tables/graphs (such as header, description, footnotes or disclaimers). For missing values or skipped cells, always use a placeholder like "n.a.". Use Markdown format for the table(s), only reply with markdown. OK?`,
+        content: `I have a PDF where I think there might be a couple of tables related to a company's CO2 emissions. Can you extract the text from screenshot if there are any? I will send you the screenshot extract the header and table contents and ignore the surrounding text if they are not related to the tables/graphs (such as header, illustrations, description, footnotes or disclaimers). For missing values or skipped cells, use a placeholder like "n.a.". Use Markdown format for the table(s), only reply with markdown. If there are no tables, just answer with an empty response. OK?`,
       },
       {
         role: 'assistant',
         content:
-          'Sure. Sounds good. Send the screenhot and I will extract the table(s) and return in markdown format as accurately as possible without any other comment.',
+          'Sure. Sounds good. Send the screenhot and I will extract the table(s) if there are any and return in markdown format as accurately as possible without any other comment.',
       },
       {
-        role: 'assistant',
+        role: 'user',
```

Comment:
always a good practice to send data about context from the user- not the assistant.